doc/changelog.rst

```diff
@@ -19,6 +30,7 @@ Backwards incompatible changes
 TODO: add examples of how to deal with these changes
 
 -  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+-  Use ``iris.FUTURE.save_split_attrs = True`` to remove iris warning (:pull:`2398`) by :user:`schlunma`
```

Comment:
@rebeccaherman1 and @schlunma, would it be possible for you to provide some examples of how to deal with these backwards incompatible changes, please? 😊

Reply:
And also upgrade advice for the deprecations please

Reply:
Thanks @schlunma! 🎉 I have added this 4f61b8e35, see also https://esmvaltool--2462.org.readthedocs.build/projects/ESMValCore/en/2462/changelog.html#backwards-incompatible-changes (when it has finished building) 👍

Reply:
I haven't heard from @rebeccaherman1, so I was wondering whether the reviewer (@valeriupredoi) would have the knowledge to provide some details for this? 🤞

Reply:
Thanks @schlunma! Could you please also add upgrading advice for #2231, which is listed under deprecations?

Reply:
Just did this via 8c492056d 😁

Reply:
Thanks!